### PR TITLE
Rework documentation structure

### PR DESCRIPTION
**Pull request by sondrelg**
Creates reusable rst-components we can include in our readthedocs files and readme's. 

We've previously maintained one root dir `README.rst` (now `README.md`), one `PYPI_README.rst`, and one `index.rst`, all with more or less the same content. This way we only have to write and update docs one place.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,20 +92,6 @@ responses:
...
-          items:
-            type: object
-            properties:
-              id:
-                type: integer
-                format: int64
-              name:
-                type: string
-            xml:
-              name: Tag
```
Comment (sondrelg) [inline README.md:106]: I removed the `tags` section of the readme toy example. It's still really long, but was **really** long

File: docs/pypi/OLD_README.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+``django-swagger-tester`` has been renamed to ``django-openapi-tester``.
+
+If you're already using ``django-swagger-tester``, please update your requirements, and if you're looking for documentation, please see https://pypi.org/project/django-openapi-tester/ instead.
```
Comment (sondrelg) [inline docs/pypi/OLD_README.rst:3]: For the `django-swagger-tester` pypi readme. This relates to #147

**Review by sondrelg — COMMENTED** [review]
Locally the readthedocs docs look the same before and after. ReadTheDocs also doesnt care where the docs are located, so think this should be pretty safe.

From their docs
```
Where do I need to put my docs for RTD to find it?

Read the Docs will crawl your project looking for a conf.py. 
Where it finds the conf.py, it will run sphinx-build in that directory. 
So as long as you only have one set of sphinx documentation in your project, it should Just Work.
```

**Review by JonasKs — APPROVED** [review]
Mmmmm, clean!

**Pull request merged**